src/java/com/sarality/app/view/list/ListViewInitializer.java

```diff
@@ -64,4 +65,8 @@ public void setupActions() {
       return new ListViewAdapter<T>(getContext(), dataList, getRowRenderer());
     }
   }
+
+  public ListViewAdapter<T> getAdapter() {
```

Comment:
Need to get the filter from the adapter. Filter is exposed but not adapter.
On 02-Mar-2015 1:42 pm, "Abhideep Singh" <notifications@github.com> wrote:

> In src/java/com/sarality/app/view/list/ListViewInitializer.java
> <https://github.com/sarality/appblocks/pull/259#discussion_r25582828>:
>
> > @@ -64,4 +65,8 @@ public void setupActions() {
> >        return new ListViewAdapter<T>(getContext(), dataList, getRowRenderer());
> >      }
> >    }
> > +
> > +  public ListViewAdapter<T> getAdapter() {
>
> why do we need to expose this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sarality/appblocks/pull/259/files#r25582828>.
>